routes/users.js

```diff
@@ -21,25 +20,56 @@ router.get("/", (req, res) => {
 
 router.get("/:id", (req, res) => {
 
-    // get a user with ID of 5bc64abcd71db3783d7e8cb2
     User.findById(req.params.id, function(err, user) {
         if (err) throw err;
 
-        // show the one user
         return res.json({ success: true, data: user } );
     });
 
 
 });
 
 
+router.post("/", (req, res) => {
+
+    let user = new User();
+
+    user.name = req.body.name;
+    user.age = req.body.age;
+
+    user.save(function(err) {
+
+        if (err) throw err;
+
```

Comment:
I guess you're returning res.json in any case. Try to return earlier: `if (err) return ...`